com.github.edenalencar.identifications.json

```diff
@@ -0,0 +1,38 @@
+{
+    "app-id" : "com.github.edenalencar.identifications",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "identifications",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "identifications",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/edenalencar/identifications.git"
```

Comment:
This needs a commit or even better use a tarball if you have one

Reply:
I already sent it to the flathub branch and my repository.
It's ok or have i to do another thing?

Reply:
not sure what you mean by "sent it to the flathub branch" but yes a commit, tag + commit is necessary. Or a release tarball.

Reply:
Done! I created tag 0.1.1 + commit with this corretion.

Reply:
I don't see it here.

Reply:
The source here should contain a tag/commit in addition of the url

Reply:
Sorry, as it is my first application submission process, I didn't understand where to generate the tag.
So I generated the 0.1.1 tag in the application repository and generated the 0.1.1 tag in the repository that I cloned to make the pull request.
Is that it or am I still not getting it?

Reply:
You to add here a `"tag": ""`, and `"commit": ""` otherwise how do you want to tell which code to build. 
Using the default branch isn't an option on flathub.

See https://docs.flatpak.org/en/latest/manifests.html#modules

Reply:
Done. I added the tag and the commit to the manifest.